src/pages/components/card-card/CartCard.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import '../card-card/cartcard.css'
+const CartCard = ({product}) => {
+    console.log(product);
+  return (
+    <div className='cartcard'>
+      <div className="img-section">
+        <img src={product.img} alt="" />
+      </div>
+      <div className="info-section-cart">
+        <div className="product-title-cart">
+        {product.title}
+        </div>
+        <div className="details-cart">
+          <div className="qty-cart">
+            <div className="minusbutton qty-btn">
```

Comment:
You can use button here and style it accordingly. it provides default attrs like disables etc.

Reply:
agreed, will change in next version.